policyengine_us/parameters/gov/states/mt/tax/income/main/capital_gains_tax_rate/rate_above_threshold_income_difference.yaml

```diff
@@ -0,0 +1,21 @@
+description: Montana provides tax for net long-term capital gains above the difference of threshold and nonqualified income at this rate, based on filing status.
+metadata:
+  label: Montana tax rate for capital gains above difference of threshold and nonqualified income
+  period: year
+  unit: currency-USD
+  breakdown:
+    - filing_status
+  reference:
+    - title: 2023 Montana Form 2 Individual Income Tax Instructions, Net Long-Term Capital Gains Tax Table
```

Comment:
I dont see the rate here

Reply:
it's in this table
![image](https://github.com/PolicyEngine/policyengine-us/assets/97819588/d8d1a182-2e12-4758-a921-5ab9f8f9430c)